cloud-provider-vsphere.yaml

```
@@ -0,0 +1,31 @@
+package:
+  name: cloud-provider-vsphere
+  version: "1.33.0"
+  epoch: 0
+  description: Kubernetes Cloud Provider for vSphere
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 41e82a9c3d926da4ed4ee294b17c6bee6bbef012
+      repository: https://github.com/kubernetes/cloud-provider-vsphere
+      tag: v${{package.version}}
+
+  - uses: go/build
+    with:
+      ldflags: -X main.version=${{package.version}}
+      output: vsphere-cloud-controller-manager
+      packages: ./cmd/vsphere-cloud-controller-manager
+
+update:
+  enabled: true
+  github:
+    identifier: kubernetes/cloud-provider-vsphere
+    strip-prefix: v
+
+test:
+  pipeline:
+    - runs: |
+        vsphere-cloud-controller-manager --help
```

Given the image is explicit with full-path to '/bin/vsphere-cloud-controller-manager', lets add a test / catch here:

```suggestion
        vsphere-cloud-controller-manager --help
        /bin/vsphere-cloud-controller-manager --help
```